From d97ef278b52a830db0fd309eeb155c954f430d60 Mon Sep 17 00:00:00 2001
From: Hao Bui <hao.bui.yg@renesas.com>
Date: Thu, 5 Nov 2020 17:50:21 +0700
Subject: [PATCH 434/446] ARM: dts: r8a7742-iwg21d-q7: Enable touch interrupt
 via gpio-hog

Enable touch panel interrupt GPIO (GPIO_0_24) via gpio-hog on the iWave
RZ/G1H (iwg21d) board.

Signed-off-by: Duy Bao Dang <duy.dang.yb@renesas.com>
Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a7742-iwg21d-q7.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7742-iwg21d-q7.dts b/arch/arm/boot/dts/r8a7742-iwg21d-q7.dts
index 4c75a2d..b2ef090 100644
--- a/arch/arm/boot/dts/r8a7742-iwg21d-q7.dts
+++ b/arch/arm/boot/dts/r8a7742-iwg21d-q7.dts
@@ -285,6 +285,14 @@
 	status = "okay";
 };
 
+&gpio0 {
+	touch-interrupt {
+		gpio-hog;
+		gpios = <24 GPIO_ACTIVE_LOW>;
+		input;
+	};
+};
+
 &gpio1 {
 	can-trx-en-gpio{
 		gpio-hog;
-- 
2.7.4

